Calibration/PPSAlCaRecoProducer/test/hlt_out_singleRAW.py

```diff
@@ -0,0 +1,293 @@
+    moduleLabelPatternsToMatch = cms.vstring( 'hlt*' ),
+    moduleLabelPatternsToSkip = cms.vstring(  )
+)
+process.hltTriggerSummaryRAW = cms.EDProducer( "TriggerSummaryProducerRAW",
+    processName = cms.string( "@" )
+)
+process.hltPreALCAPPSOutput = cms.EDFilter( "HLTPrescaler",
+    offset = cms.uint32( 0 ),
+    L1GtReadoutRecordTag = cms.InputTag( "hltGtStage2Digis" )
+)
+
+process.hltOutputALCAPPS = cms.OutputModule( "PoolOutputModule",
+    fileName = cms.untracked.string( "outputALCAPPS_single.root" ),
+    fastCloning = cms.untracked.bool( False ),
+    dataset = cms.untracked.PSet(
+        filterName = cms.untracked.string( "" ),
+        dataTier = cms.untracked.string( "RAW" )
+    ),
+    SelectEvents = cms.untracked.PSet(  SelectEvents = cms.vstring( 'HLT_PPSMaxTracksPerArm1_v1',
+  'HLT_PPSMaxTracksPerRP4_v1' ) ),
+    outputCommands = cms.untracked.vstring( 
+      'keep *_hltPPSCalibrationRaw_*_*',
+      'keep *_hltGtStage2ObjectMap_*_*',
+      'keep edmTriggerResults_*_*_*',
+      'keep triggerTriggerEvent_*_*_*' )
+)
+
+process.HLTL1UnpackerSequence = cms.Sequence( process.hltGtStage2Digis + process.hltGtStage2ObjectMap )
+process.HLTBeamSpot = cms.Sequence( process.hltScalersRawToDigi + process.hltOnlineBeamSpot )
+process.HLTBeginSequence = cms.Sequence( process.hltTriggerType + process.HLTL1UnpackerSequence + process.HLTBeamSpot )
+process.HLTPPSPixelRecoSequence = cms.Sequence( process.hltCTPPSPixelDigis + process.hltCTPPSPixelClusters + process.hltCTPPSPixelRecHits + process.hltCTPPSPixelLocalTracks )
+process.HLTEndSequence = cms.Sequence( process.hltBoolEnd )
+
+process.HLTriggerFirstPath = cms.Path( process.hltGetConditions + process.hltGetRaw + process.hltPSetMap + process.hltBoolFalse )
+process.HLT_PPSMaxTracksPerRP4_v1 = cms.Path( process.HLTBeginSequence + process.hltL1sZeroBias + process.hltPrePPSMaxTracksPerRP4 + process.HLTPPSPixelRecoSequence + process.hltPPSPrCalFilter + process.hltPPSCalibrationRaw + process.HLTEndSequence )
+process.HLT_PPSMaxTracksPerArm1_v1 = cms.Path( process.HLTBeginSequence + process.hltL1sZeroBias + process.hltPrePPSMaxTracksPerArm1 + process.HLTPPSPixelRecoSequence + process.hltPPSExpCalFilter + process.hltPPSCalibrationRaw + process.HLTEndSequence )
+process.HLTriggerFinalPath = cms.Path( process.hltGtStage2Digis + process.hltScalersRawToDigi + process.hltFEDSelector + process.hltTriggerSummaryAOD + process.hltTriggerSummaryRAW + process.hltBoolFalse )
+process.ALCAPPSOutput = cms.EndPath( process.hltGtStage2Digis + process.hltPreALCAPPSOutput + process.hltOutputALCAPPS )
+
+
+process.HLTSchedule = cms.Schedule( *(process.HLTriggerFirstPath, process.HLT_PPSMaxTracksPerRP4_v1, process.HLT_PPSMaxTracksPerArm1_v1, process.HLTriggerFinalPath, process.ALCAPPSOutput, ))
+
+
+process.source = cms.Source( "PoolSource",
+    fileNames = cms.untracked.vstring(
+        'file:RelVal_Raw_GRun_DATA.root',
+    ),
+    inputCommands = cms.untracked.vstring(
+        'keep *'
```

Comment:
Normally scary for the event size... but ok, it is a test script...

Reply:
Note that this is Source's `inputCommands`, i.e. for dropping some branches on input. Single `keep *` is redundant though, and for clarity the whole `inputCommands` PSet could be dropped.